src/server/protocol.ts

```diff
@@ -423,7 +423,7 @@ export interface OutliningSpansRequestFull extends FileRequest {
  *
  * @internal
  */
-export interface OutliningSpansResponseFull extends Response {
+interface OutliningSpansResponseFull extends Response {

```

Comment:
Is there a canonical way to tell knip to ignore things? This is the protocol and likely we don't want to drop things in here, though being internal is a little funky.

Reply:
Yes: https://knip.dev/guides/handling-issues#unused-exports (tl/dr; export it from an entry file or add `@public` tag)